src/renderer/state.ts

```diff
@@ -128,7 +129,8 @@ export class AppState {
       : (this.retrieve('executionFlags') as Array<string>);
   @observable public packageManager: IPackageManager =
     (localStorage.getItem('packageManager') as IPackageManager) || 'npm';
-
+  @observable public acceleratorsToBlock: Array<BlockableAccelerator> =
```

Comment:
Working with a Set here, would require custom handling of Sets in the `save` method in `state.ts`. Since a `Set` would qualify as `object` and _stringifying_ a set imho does not work and has it has to be converted to an array in this case anyway.
I am open if custom set handling would be preferred.

Reply:
@k0rn1 you're right on that one. We could keep it as an array for simplicity's sake. Will resolve this!